File: integration-tests/hibernate-reactive-panache/src/main/java/io/quarkus/it/panache/reactive/CatOwner.java
package io.quarkus.it.panache.reactive;

import jakarta.persistence.Entity;

import io.quarkus.hibernate.reactive.panache.PanacheEntity;

@Entity
public class CatOwner extends PanacheEntity {

    public String name;

    public CatOwner(String name) {
        this.name = name;
    }

    public CatOwner() {
    }
}

File: integration-tests/hibernate-reactive-panache-kotlin/src/main/kotlin/io/quarkus/it/panache/reactive/kotlin/CatOwner.kt
package io.quarkus.it.panache.reactive.kotlin

import io.quarkus.hibernate.reactive.panache.kotlin.PanacheCompanion
import io.quarkus.hibernate.reactive.panache.kotlin.PanacheEntity
import jakarta.persistence.Entity

@Entity
class CatOwner @JvmOverloads constructor(var name: String? = null) : PanacheEntity() {
    companion object : PanacheCompanion<CatOwner>
}
